### PR TITLE
Added Disable schedule and Enabled/Disabled filter.

### PR DESCRIPTION
**Pull request by Jonsy13**
Signed-off-by: Jonsy13 <vedant.shrotria@chaosnative.com>

## Proposed changes

Summarize your changes here to communicate with the maintainers and make sure to put the link of that issue

This PR will add disable schedule and Enabled/Disabled Filter for schedules table.

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [X] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [X] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [X] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

File: litmus-portal/frontend/src/views/ChaosWorkflows/BrowseSchedule/index.tsx [hunk 1]
```diff
@@ -154,6 +205,30 @@ const BrowseSchedule: React.FC = () => {
...
+                  ...filter,
+                  suspended: event.target.value as string,
+                })
+              }
+              label="Name"
+              className={classes.selectText}
+            >
+              <MenuItem value="All">Show All Workflows</MenuItem>
+              <MenuItem value="false">Show Enabled Workflows Only</MenuItem>
+              <MenuItem value="true">Show Disabled Workflows Only</MenuItem>
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/views/ChaosWorkflows/BrowseSchedule/index.tsx:228]: Translations are missing

**Review by ishangupta-ds — CHANGES_REQUESTED** [review]
@Jonsy13 PTAL

File: litmus-portal/frontend/src/views/ChaosWorkflows/BrowseSchedule/index.tsx (hunk 1 above)
Comment (Jonsy13) [reply litmus-portal/frontend/src/views/ChaosWorkflows/BrowseSchedule/index.tsx]: Thanks @ishangupta-ds , PTAL

**Review by S-ayanide — APPROVED** [review]

**Review by amityt — APPROVED** [review]

**Review by ishangupta-ds — APPROVED** [review]
LGTM

**Pull request merged**